.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 .DS_Store
+*.code-workspace
```

Comment:
I had to configure a multi-root code space to get rust-analyzer to work with all of the crates but I can also remove this .gitignore change if you think it's noisy, let me know.

Reply:
Yeah don't worry this is fine 🙂